packages/curve-ui-kit/src/shared/ui/LargeTokenInput.tsx

```diff
@@ -149,6 +149,7 @@ export type LargeTokenInputProps = {
    */
   tokenSelector?: ReactNode
 
+  // TODO: rename to just "balance" because multipurpose now (walletBalance, positionBalance ...)
```

Comment:
yeah there is already a `balance` prop
```suggestion
  // TODO: receive a `maxBalance` ReactNode to allow anything to be injected
```